src/plugins/embeddable/public/lib/triggers/triggers.ts

```diff
@@ -27,10 +27,7 @@ export interface EmbeddableContext {
 export interface EmbeddableVisTriggerContext {
   embeddable?: IEmbeddable;
   timeFieldName?: string;
-  data: {
-    e?: MouseEvent;
-    data: unknown;
-  };
+  data: unknown;
```

Comment:
https://github.com/elastic/kibana/issues/62972

Reply:
this is related, but on the side of expressions. this PR is adding types on the level of actions. On the top level trigger type the data is `unknown` as we do not know specific trigger that emitted the data. As soon as we know which trigger emitted the data we can cast it to correct trigger type and we get full typing of it.

could we in dashboard to dashboard drilldown do this ? based on the trigger that was emitted cast the context into correct type ? i guess at the moment action does not get information which of the possible triggers triggered it. should we provide this information to actions ?

anyway, i would leave this out of this PR.

Reply:
OK, will do the casting in dashboard-to-dashboard pr